src/validator.js

```diff
@@ -446,14 +446,15 @@ const validator = {
     options = options || {};
     documentedHooks = documentedHooks || [];
     hooksContainer = hooksContainer || {};
-    documentedHooks.forEach(function (hook) {
+    for (let i = 0; i < documentedHooks.length; i++) {
+      let hook = documentedHooks[i];
       const hookType = typeof(hooksContainer[hook]);
       const allowedTypes = ['function', 'undefined'];
       if (!allowedTypes.includes(hookType)) {
         helpers.throwError(options, 'The ' + location + hook + ' must be a function or undefined.');
         delete hooksContainer[hook];
       }
-    });
+    }
```

Comment:
There are 4 `forEach` in this file, 3 still need converted.

Reply:
Yes those 3 has static array data e.g `['styles', 'markup', 'scripts']` 
should I store it to a variable and then make it `for` loop?

Reply:
Nevermind, I have made the required changes to those 3 loops aswell 👍

Reply:
Oh yeah, you're right, we don't really need those to be traditional for loops. I was just doing a `ctrl+f` across the code and didn't check that. Probably cleaner to revert them, like you had it originally, then we can merge this

Reply:
OK great, reverting back